startup.sh

```diff
@@ -6,6 +6,16 @@ if [ $? -ne 0 ]; then
   echo "You need to install docker on this machine before you can install Engelsystem/FC."
 fi
 
+compose() {
+  if command -v docker-compose >/dev/null 2>&1; then
+    # Use docker-compose if it's installed
+    docker-compose "$@"
```

Comment:
I wrote the following test in `/bin/sh`

```
 foobar() {
    echo "$0"
}
```

Then I typed `foobar build --no-cache`

It did not echo `build --no-cache`.

Reply:
Did you mean to type `$0` (dollarsign zero), or `$@` (dollarsign atsign) like my code? 

https://linuxhandbook.com/bash-special-variables/

Reply:
Aha. I mistyped. That explains why my test wasn't working. Thank you!